stdlib/REPL/test/repl.jl

```diff
@@ -1033,3 +1033,90 @@ fake_repl() do stdin_write, stdout_read, repl
     wait(repltask)
     @test istaskdone(repltask)
 end
+
+# shell escaping on Windows
+
+
+@testset "shell_escape_winsomely" begin
+# Note  argument A can be parsed both as A or "A".
+# We do not test that the parsing satisfies either of these conditions.
+# In other words, tests may fail even for valid parsing.
+# This is done to avoid overly verbose tests.
+
+# input :
+# output: ""
+@test Base.shell_escape_winsomely("") == "\"\""
```

Comment:
Is this the best place to put the tests? They are not testing anything of the REPL code.

It seems other tests are in `test/spawn.jl`:

https://github.com/JuliaLang/julia/blob/fa235cc83ad13c301e1242ed1654429beaa5cb53/test/spawn.jl#L402-L414

so maybe that is better? Also, I think the block after a testset tends to be indented.

Reply:
Thanks for the suggestion . Totally agree and will update the PR.